test/recipes/80-test_ssl_old.t

```diff
@@ -445,6 +443,8 @@ sub testssl {
                 $flag = "-ssl3";
             } elsif ($protocol eq "TLSv1.2") {
                 $flag = "-tls1_2";
+            } elsif ($protocol eq "TLSv1.3") {
```

Comment:
Okie, sounds good. Should we explain why we don't have flags for tls 1.0 and 1.1?